OpenEXR/IlmImfExamples/lowLevelIoExamples.cpp

```diff
@@ -98,7 +98,7 @@ class C_OStream: public OStream
 bool
 C_IStream::read (char c[/*n*/], int n)
 {
-    if (n != fread (c, 1, n, _file))
+    if (size_t(n) != fread (c, 1, n, _file))
```

Comment:
maybe prefer static_cast on rhs? and below

Reply:
done